packages/core/src/diff/node.rs

```diff
@@ -328,6 +328,9 @@ impl VNode {
                     to.remove_node(id);
                 }
                 dom.reclaim(id);
+            } else {
+                let id = dom.get_mounted_root_node(mount, idx);
```

Comment:
Reclaiming the root mounted node here when mutations is None seems odd. If this is a suspended node, that id should never have been set [here](https://github.com/DioxusLabs/dioxus/blob/797e7492a1b9e188627ab7530a66607ba19376d6/packages/core/src/diff/node.rs#L607-L609)